app/open_ai/text_events.py

```
@@ -0,0 +1,44 @@
+            }
+        ],
+        temperature=0.2
+    )
+    gpt_output = response.choices[0].message.content
+
+    if not gpt_output:
+        raise ValueError(f"Output of gpt is None.\nig_text was:{ig_text}")
+    
+    try:
+        event_dict = json.loads(gpt_output)
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)